Express/controller.js

```diff
@@ -48,10 +49,23 @@ const getCommentsByArticleID = async (req, res, next) => {
   }
 };
 
+const postComment = async (req, res, next) => {
+  try {
+    const comment = req.body;
+    const { article_id } = req.params;
+    const newComment = await uploadNewComment(article_id, comment);
+    console.log(newComment);
```

Comment:
Dont forget to remove console.log's/comments that are no longer needed before pushing up to github